recipes/nexus/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "nexus" %}
+{% set version = "4.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nexusformat/code/archive/v{{ version }}.tar.gz
+  sha256: e78a116feb2ebd04de31a8d8707c65e8e15a64aa8999a40fea305e3909bd6533
+
+build:
+  number: 0
+  skip: true    # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake >=2.8.8
+  host:
+    - hdf5
+    - hdf4
+    - jpeg
+    - libxml2
+    - ncurses
+    - readline
+    - zlib
+  run:
+    - hdf5
+    - hdf4
+    - jpeg
```

Comment:
no need for jpeg. it includes a run exports
https://github.com/conda-forge/jpeg-feedstock/blob/master/recipe/meta.yaml#L15